srcpkgs/raylib/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'raylib'
+pkgname=raylib
+version=2.6.0
+revision=1
+archs="i686 x86_64"
+build_style=cmake
+configure_args="-D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=/usr -D CMAKE_INSTALL_LIBDIR=lib -D OpenGL_GL_PREFERENCE=GLVND -D USE_EXTERNAL_GLFW=ON -D BUILD_EXAMPLES=OFF -D BUILD_GAMES=OFF -D SHARED=ON -D STATIC=ON"
```

Comment:
This can be reduced, the first items are already set by `build_style=cmake`. Also, split the line and indent the second line with a space.

    configure_args="-D OpenGL_GL_PREFERENCE=GLVND -D USE_EXTERNAL_GLFW=ON 
     -D BUILD_EXAMPLES=OFF -D BUILD_GAMES=OFF -D SHARED=ON -D STATIC=ON"